dev_tools/generate_documentation.rst

```diff
@@ -29,30 +29,24 @@ On Ubuntu (or other Debian-based distributions), the relevant commands can be cu
 
 There are two files which may require modifications, on top of the other generated files (html, json, ...)
 
-- `README.rst`: A README update for users visiting the Github page.
+- `README.rst`: A README update for users visiting the GitHub page.
 - `index.qmd`: The main page of the rendered documentation, listing and tying together all the other rendered html files.
 
-2b. Regenerate the html files
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+2b. Locally regenerating the html files (optional)
```

Comment:
Let's remove the "optional" and say in the section

This step regenerates the html documentation locally to ensure notebooks render as they should, and that the navigation works as expected.